test/unit-test/framework/mock_securechip.c

```diff
@@ -36,3 +38,19 @@ bool securechip_kdf(securechip_slot_t slot, const uint8_t* msg, size_t len, uint
     memcpy(kdf_out, (const uint8_t*)mock(), 32);
     return true;
 }
+
+bool securechip_u2f_counter_inc(uint32_t* counter)
+{
+    *counter = _u2f_counter++;
+    return true;
+}
+
+bool securechip_ecc_unsafe_sign(const uint8_t* priv_key, const uint8_t* msg, uint8_t* sig)
+{
+    return false;
```

Comment:
do the tests not exercise signing?

Reply:
not right now, I guess I could mock it somehow, but I can wait with that.